internal/DynamicParams.ps1

```diff
@@ -3,10 +3,10 @@
 	These are all the functions for tab completion (auto-population of params)
 	To use, place this after params in a function
 
-	DynamicParam { if ($source) { return (Get-ParamSqlXyz -SqlServer $Source -SqlCredential $SourceSqlCredential) } }
+	DynamicParam { if ($source) { return (Get-ParamSqlXyz -SqlInstance $Source -SqlCredential $SourceSqlCredential) } }
 
 #>
-Function Get-ParamSqlServerConfigs
+Function Get-ParamSqlInstanceConfigs
```

Comment:
Not seeing any references to the functions renamed in this, this might be my understanding of how dynamic params work though, but I would confirm that all the function renames were what is needed as there are several in this file.